Matrix.py

```diff
@@ -10,18 +7,12 @@ class Matrix:
             data(tuple of tuples): a Matrix- each line is a tuple in the major tuple """
 
     def __init__(self, data):
-        #  inits a Matrix from received data (type= tuple), throws error for non- tuple inputs.
-        check_value = True
-        if type(data) is tuple:
-            for tup in data:
-                if len(tup) != len(data) or type(tup) is not tuple:
-                    check_value *= 0
-        else:
-            check_value *= 0
-        if check_value:
-            self.data = data
-        else:
+        if type(data) is not tuple:
             raise ValueError("Invalid matrix")
```

Comment:
what about blank like after logic block, please fix it everywhere and don't repeat it :/